README.md

```diff
@@ -115,13 +115,57 @@ public void ConfigureServices(IServiceCollection services)
             // For demo purposes only. Private key should be stored securely
             options.Payments.SigningKey.PrivateKey = File.ReadAllText("ec512-private-key.pem");
         }
-    })
-    // We advice to cache the auth token
-    .AddAuthTokenInMemoryCaching();
+    },
+    // For best performance and reliability we advice to cache the auth token
+    authCachingStrategy: AuthCachingStrategy.InMemory)
+
 }
 ```
 
-Alternatively you can create a class that implements `IConfigureOptions<TrueLayerOptions>` if you have more complex configuration requirements.
+### Multiple TrueLayer Clients
+
+Use keyed version of TrueLayer client (.NET 9.0/.NET 8.0):
+
+```c#
+.AddKeyedTrueLayer(configuration, options =>
+    {
+        // For demo purposes only. Private key should be stored securely
+        var privateKey = File.ReadAllText("ec512-private-key.pem");
+        if (options.Payments?.SigningKey != null)
+        {
+            options.Payments.SigningKey.PrivateKey = privateKey;
+        }
+    },
+    configurationSectionName: "TrueLayerGbp",
+    serviceKey: "TrueLayerGbp",
```

Comment:
Do both need to be configured? Can we look for the `serviceKey` by default? And use the `configurationSectionName` only if explicitly provided?

Reply:
yeah we could do that